src/ports/postgres/modules/knn/test/knn.sql_in

```diff
@@ -0,0 +1,41 @@
+m4_include(`SQLCommon.m4')
+/* -----------------------------------------------------------------------------
+ * Test knn.
+ *
+ * FIXME: Verify results
```

Comment:
Actually I don't know how this file is gonna used by user. I observed other modules to create this file. Please elaborate.